### ISSUE TITLE
Incorrect error handling in case of broken permissions to whisper file

### ISSUE THREAD

**Issue by pavel-odintsov** [go-graphite/go-carbon#21]
Hello!

I have changed user for go-carbon from root to _graphite and fit interesting bug.

Actually files stored on disk with root permissions and go-carbon could not read it from user _graphite. 

It's OK but it tried to create new files:

``` bash
[2015-11-26 13:49:14] E [persister] Failed to create new whisper file /var/lib/graphite/whisper/carbon/agents/netflow_fv_ee/cache/checkpointTime.wsp: file already exists
[2015-11-26 13:49:14] E [persister] Failed to create new whisper file /var/lib/graphite/whisper/carbon/agents/netflow_fv_ee/cache/overflow.wsp: file already exists
[2015-11-26 13:49:14] E [persister] Failed to create new whisper file /var/lib/graphite/whisper/carbon/agents/netflow_fv_ee/cache/queries.wsp: file already exists
[2015-11-26 13:49:14] E [persister] Failed to create new whisper file /var/lib/graphite/whisper/carbon/agents/netflow_fv_ee/cache/metrics.wsp: file already exists
[2015-11-26 13:49:14] E [persister] Failed to create new whisper file /var/lib/graphite/whisper/carbon/agents/netflow_fv_ee/cache/size.wsp: file already exists
```

So I suggest to report this sort of error in more detailed way and do not try to create file in this case.

Thanks!

**Comment by lomik** [comment]
Fixed in master https://github.com/lomik/go-carbon/commit/cb2724eb606b02c5111f122bb39af4b0bd93a1ec

**Comment by pavel-odintsov** [comment]
Nice! Thanks!

On Thursday, 5 May 2016, Roman Lomonosov notifications@github.com wrote:

> Closed #21 https://github.com/lomik/go-carbon/issues/21.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/lomik/go-carbon/issues/21#event-651855549

## 

Sincerely yours, Pavel Odintsov
